era5cli/utils.py

```diff
@@ -211,14 +212,17 @@ def strtobool(value: str) -> bool:
 def assert_outputfiles_not_exist(outputfiles: List[str]) -> None:
     """Check if files already exist, and prompt the user if they do."""
     if any(Path(file).exists() for file in outputfiles):
-        answer = input(
-            "\n  Some file(s) that will be downloaded already exist in this folder."
-            "\n  Do you want to overwrite them? (Y/N)"
-            "\n  Tip: to skip this flag, use `--overwrite`."
-            "\n"
-        )
+        answer = "no"  # default answer for non-interactive sessions
+        if sys.stdin.isatty():  # only ask for input if the user can reply.
+            answer = input(
+                "\n  Some file(s) that will be downloaded already exist in this folder."
+                "\n  Do you want to overwrite them? (Y/N)"
+                "\n  Tip: to skip this flag, use `--overwrite`."
+                "\n"
+            )
         if answer.lower() in ["n", "no", "nope"]:
```

Comment:
If 'answer' is '' (just press enter) then file is overwritten. I was expecting no as the default.